sites/public/src/patterns/CardList.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { Heading, Card } from "@bloom-housing/ui-seeds"
+import styles from "./CardList.module.scss"
+
+interface ContentCardProps {
+  title: string
+  description?: React.ReactNode
+  priority?: 1 | 4 | 2 | 3 | 5 | 6
```

Comment:
question: how come the type here is a random order of the number 1 - 6?

Reply:
This is a type from Seeds (that def shouldn't be out of order) - I'd love to export the type in Seeds and pull that in here vs define the type here. I'll make sure that's captured!